generators/CoreCLRStatefulActor/templates/service/app/appPackage/servicePackage/Code/entryPoint.cmd

```diff
@@ -0,0 +1,2 @@
+dotnet %~dp0\<%= serviceProjName %>.dll
```

Comment:
Let's get rid of entrypoint in Windows . I thought we discussed this

Reply:
Yeah we are not using entrypoint script for windows. We were generating it but actuallly not using it.Now its removed.